features/support/env.rb

```diff
@@ -23,6 +23,16 @@
   Delayed::Worker.delay_jobs = false
 end
 
+Before '@enable_caching' do
+  ActionController::Base.perform_caching = true
```

Comment:
as of now, `config/environments/test.rb` does not have `perform_caching` anywhere. Seems safer to explicitly test only where needed? Or maybe have test match prod and apply caching globally in the test env?